### ISSUE TITLE
Topic 10: Color Contrast Issue w/Large Text

### ISSUE THREAD

**Issue by HD508Github** [Section508Coordinators/DHSA-TT-200-A#19]
Lesson4-Lists > Failing Example > Example 2 -- Image Large text fails Color Contrast.
![Top10_CCA Issue (Lesson4-Failing xamples page)](https://user-images.githubusercontent.com/34749031/58511684-4a52cb80-8169-11e9-96b0-83375f126bc2.jpg)

**Comment by KSmithO** [comment]
Questions: do we really need this fixed (headings have no impact on purpose of the image and convey no real meaning for image purpose)? if we want this fixed, how much of an effort would it be?

**Comment by HD508Github** [comment]
Given that rationale, I would say let it go.

Kind Regards,
Halimatou(Halima)Diallo
Sr. Accessibility Technician
Supporting the Department of Homeland Security|MGMT/OCIO/OAST
Office: 202-815-5957
Email: Halimatou.Diallo@associates.hq.dhs.gov<mailto:Halimatou.Diallo@associates.hq.dhs.gov>
DHS Accessibility Help Desk: 202-447-0440 (Voice), 202-447-0582 (Fax)
Website: accessibility.dhs.gov<http://accessibility.dhs.gov/> | Email: accessibility@hq.dhs.gov<mailto:accessibility@hq.dhs.gov>
[cid:image001.png@01D3A09F.E6927B00]
This communication, along with any attachments, is covered by federal and state law governing electronic communications and may contain sensitive and legally privileged information. If the reader of this message is not the intended recipient, you are hereby notified that any dissemination, distribution, use or copying of this message is strictly prohibited.  If you have received this message in error, please reply immediately to the sender and delete this message.  Thank you.

**Comment by drewnielson** [comment]
The text that accompanies the image also describes in detail the pertinent information contained in the image. With that text, there is no loss of information for any user. I recommend not spending the time to revise the image. Closing the issue, but we can re-open if someone has a strong objection.
